api/exec/util/get_workspace_info.go

```
@@ -0,0 +1,25 @@
+
+func GetWorkspaceId() (string, error) {
+	rawWorkspaceInfo, err := os.ReadFile("./dcd-info/workspace-info.json")
+	if err != nil {
+		return "", errors.New("워크스페이스 정보를 찾을수없습니다.")
+	}
+
+	var workspaceInfo map[string]interface{}
+
+	err = json.Unmarshal(rawWorkspaceInfo, &workspaceInfo)
+	if err != nil {
+		return "", errors.New("워크스페이스 정보가 올바르지 않습니다.")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not end with punctuation or newlines

(staticcheck)